setup.py

```diff
@@ -17,15 +17,15 @@
     long_description = readme.read()
 
 validate_dependencies = [
-    "pytest>=6.2.5,<7",
+    "pytest>=6.2.5",
```

Comment:
If all the tests work with pytest 7, we'd like to upgrade to 7 instead. Also, we'd like to lock the version not to automatically ugprade to 8. This SDK does not provide modules for testing, and pytest is only used for tests for this SDK itself. So, no need to have the two-major-version range here.